src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/CanvasTests/Canvas_Measurement_Tests.cs

```diff
@@ -50,5 +50,23 @@ public void Verify_Canvas_With_Outer_Clip()
 
 			AssertHasColorAt(screenshot, unclippedLocation.CenterX, unclippedLocation.CenterY, Color.Blue);
 		}
+
+		[Test]
+		[AutoRetry]
+		[ActivePlatforms(Platform.Android, Platform.iOS)] // Canvas.ZIndex isn't implemented for WASM yet
+		public void Verify_Canvas_ZIndex()
```

Comment:
Are you able to check the color on the outside edge of the rectangle and be sure there's no shadow ? (if there's a shadow it will be green+some partial gray)

Reply:
Good idea!